astroquery/esa/xmm_newton/tests/test_xmm_newton.py

```diff
@@ -295,3 +298,46 @@ def test_get_epic_images(self):
         for ob_name in self._files:
             shutil.rmtree(ob_name)
         os.remove(_tarname)
+
+    @pytest.mark.remote_data
```

Comment:
now this is fine, but as a follow-up, the remote_data test could be factored out into a separate test file to follow what is done in other modules.